docs/transparent-validation/sd-wire/usage-validation.md

```diff
@@ -2,109 +2,190 @@
+user. 
+
+Hardware requirements:
+
+* SDWire
+* SD card
+* DUT (Device Under Test) bootable from SD card (RPI, Orange PI etc.)
+* DUT power supply
+* Micro-USB-->USB cable
+
+To perform first use procedure reproduce the following steps:
+
+1. Prepare environment in accordance with this [section](#environment-preparation).
+
+2. Insert SD card to the SDWire.
+
+3. Put SDWire into the DUT (Device Under Test) and connect it to the TS with 
+    the micro USB - USB cable.
+
+4. Check whether SDWire is configured by reproducing the following steps:
+    * run in terminal the following command:
+        ```
+        dmesg -w
+        ```
+    * connect the SDWire to your machine using micro-USB--> USB cable.
+    * after connecting your `dmesg` output should looks like this:
+        ```
+        (...)
+        [73278.307591] usb-storage 3-1.1:1.0: USB Mass Storage device detected
+        [73278.307823] scsi host6: usb-storage 3-1.1:1.0
+        [73278.384925] usb 3-1.2: new full-speed USB device number 45 using xhci_hcd
+        [73278.492025] usb 3-1.2: New USB device found, idVendor=0403, idProduct=6015, bcdDevice=10.00
+        [73278.492027] usb 3-1.2: New USB device strings: Mfr=1, Product=2, SerialNumber=3
+        [73278.492028] usb 3-1.2: Product: FT200X USB I2C
+        [73278.492029] usb 3-1.2: Manufacturer: FTDI
+        [73278.492030] usb 3-1.2: SerialNumber: DB007V7V
+        (...)
+        ```
+    * open new terminal window and run the following command:
+        ```
+        sudo  sd-mux-ctrl --list
+        ```
+        If output looks like below, it means that SDWire is configured and ready
+        to use. Now, you can go to point 5 in this section.
+        ```
+        Number of FTDI devices found: 1
+        Dev: 0, Manufacturer: SRPOL, Serial: sd-wire_11, Description: sd-wire
+        ```
+        Otherwise, if output shows no devices (like in the example below):
+        ```
+        Number of FTDI devices found: 0
+        ```
+        you have to configure SDWire:
+        ```
+        sudo sd-mux-ctrl --device-serial=DB007V7V --vendor=0x0403 --product=0x6015 --device-type=sd-wire --set-serial=sd-wire_11
+        ```
+        where:
+        ```
+        --device-serial=<SerialNumber> (from dmesg output)
+
+        --vendor=0x<idVendor> (from dmesg output)
+
+        --product=0x<idProduct> (from dmesg output)
+
+        --set-serial=<New serial device>
+        ```
+        After above-desribed procedure check again if SDWire is properly 
+        configured:
+        ```
+        sudo  sd-mux-ctrl --list
+        ```
+        Should output:
+        ```
+        Number of FTDI devices found: 1
+        Dev: 0, Manufacturer: SRPOL, Serial: sd-wire_11, Description: sd-wire
+        ```
+
+5. Connect SD card to the TS (test server):
+    ```
+    sudo sd-mux-ctrl --device-serial=sd-wire_11 --ts
+    ```
+6. Flash the SD card using `bmaptool` or balenaEtcher.
```

Comment:
@sulewskiprzemyslaw
* add example bmaptool command with the image for RPi
* add link to the balenaEtcher program DL
* add download link to the example SD image for RPi

Reply:
Description in the newest pushed version has been added.